### PR TITLE
builds url when tags are selected

### PR DESCRIPTION
**Pull request by Danwhy**
ref #321

### CODE REVIEW

**Review by samhstn — APPROVED** [review]
Looks good

File: cms/static/js/personalise.js [hunk 1]
```diff
@@ -136,4 +150,40 @@ function getOrder() {
   );
 }
 
+function updateUrl(tag) {
+  var jointTag = tag.name.split(' ').join('%20');
```
Comment (SimonLab) [inline cms/static/js/personalise.js:154]: could we use replace instead of split and join?

File: cms/static/js/personalise.js [hunk 2]
```diff
@@ -136,4 +150,40 @@ function getOrder() {
...
+  var reg = new RegExp("&*" + tag.tag_type + "=" + jointTag + "(&|$)");
+  var newUrl = window.location.href;
+  var prefix;
+
+  if (window.location.href.indexOf('?') > -1) {
```
Comment (SimonLab) [inline cms/static/js/personalise.js:159]: we could also use the ternary operator here, style preference issue

**Review by SimonLab — APPROVED** [review]
LGTM :+1: just some small suggestion that can be implemented if you want on the next PR

**Pull request merged**